awscli/testutils.py

```diff
@@ -705,3 +707,8 @@ def assert_no_errors(self, p):
         self.assertNotIn("failed:", p.stderr)
         self.assertNotIn("client error", p.stderr)
         self.assertNotIn("server error", p.stderr)
+
+
+class StringIOWithFileNo(StringIO):
+    def fileno(self):
+        return 0
```

Comment:
True, I guess it would matter more if the class was specifically for standard output. It is just being used for standard output for both cases it is used.